tests/expected/ptr_to_ref_cast/expected

```diff
@@ -0,0 +1,75 @@
+Checking harness check_zst_deref...
+
+Status: FAILURE
```

Comment:
Adding a backslash at the end, i.e.
```suggestion
Status: FAILURE\
```
ensures that the script checks that the next line in the expected file appears after the next line in the output. This is necessary to make sure that the failure is for that specific check.